internals/daemon/response.go

```diff
@@ -114,29 +114,29 @@ func (r *resp) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
 	w.Write(bs)
 }
 
-type errorKind string
+type ErrorKind string
 
 const (
-	errorKindLoginRequired     = errorKind("login-required")
-	errorKindDaemonRestart     = errorKind("daemon-restart")
-	errorKindSystemRestart     = errorKind("system-restart")
-	errorKindNoDefaultServices = errorKind("no-default-services")
-	errorKindNotFound          = errorKind("not-found")
-	errorKindPermissionDenied  = errorKind("permission-denied")
-	errorKindGenericFileError  = errorKind("generic-file-error")
+	ErrorKindLoginRequired     = ErrorKind("login-required")
+	ErrorKindDaemonRestart     = ErrorKind("daemon-restart")
+	ErrorKindSystemRestart     = ErrorKind("system-restart")
+	ErrorKindNoDefaultServices = ErrorKind("no-default-services")
+	ErrorKindNotFound          = ErrorKind("not-found")
+	ErrorKindPermissionDenied  = ErrorKind("permission-denied")
+	ErrorKindGenericFileError  = ErrorKind("generic-file-error")
 )
 
-type errorValue interface{}
+type ErrorValue interface{}
 
-type errorResult struct {
+type ErrorResult struct {
 	Message string     `json:"message"` // note no omitempty
-	Kind    errorKind  `json:"kind,omitempty"`
-	Value   errorValue `json:"value,omitempty"`
+	Kind    ErrorKind  `json:"kind,omitempty"`
+	Value   ErrorValue `json:"value,omitempty"`
```

Comment:
Similar here. This is already public in the client package already, and might be imported from there both here in and in KernOS, depending on whether the client package is already easily importable here.

Reply:
Tried this in the next proposal. Closing for further new comments to follow.